src/lab1/structures/vertex/Vertex3D.ts

```diff
@@ -11,6 +15,27 @@ export default class Vertex3D {
     this.y = y;
     this.z = z;
   }
+  translate(x: number, y: number, z: number): Vertex3D {
+    const tranformed = transformVector(
+      this.toVector(),
+      transformations.translate3d(x, y, z)
+    );
+    return tranformed.toVertex3D();
+  }
+  rotate(angleX: number, angleY: number, angleZ: number): Vertex3D {
+    const tranformed = transformVector(
+      this.toVector(),
+      transformations.rotate3d(angleX, angleY, angleZ)
+    );
+    return tranformed.toVertex3D();
+  }
+  scale(x: number, y: number, z: number): Vertex3D {
+    const tranformed = transformVector(
+      this.toVector(),
+      transformations.scale3d(x, y, z)
+    );
+    return tranformed.toVertex3D();
+  }
```

Comment:
Rotating and scaling vertex? It seems that we don't need this methods here
```suggestion
```

Reply:
No, it's used (in Sphere and Triangle at least). Rotating means rotating around the (0;0;0) point, scaling means pushing away from (0;0;0) by some factor. When you scale a Triangle, you scale every of its three vertices.